app/containers/HeaderButton/Common.js

```diff
@@ -23,10 +21,7 @@ export const CloseModal = React.memo(({
 
 export const CancelModal = React.memo(({ onPress, testID }) => (
 	<Container left>
-		{isIOS
-			? <Item title={I18n.t('Cancel')} onPress={onPress} testID={testID} />
-			: <Item iconName='close' onPress={onPress} testID={testID} />
-		}
+		<Item iconName='close' onPress={onPress} testID={testID} />
```

Comment:
It's being used on other places.
I don't think we want to change it there as well.

Reply:
Reverted